profile/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center">
+<img src="https://electricracing.northeastern.edu/wp-content/uploads/2023/02/cropped-NER-logo-white.png" alt="drawing" width="200"/>
+</p>
+
+<p align="center">
+<a href="https://electricracing.northeastern.edu">Website </a> | <a href="https://finishlinebyner.com">FinishLine </a> | <a href="https://www.linkedin.com/company/northeasternelectricracing"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/LinkedIn.svg" width="16" height="16"> </a>
+| <a href="https://www.instagram.com/nuelectricracing/"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/Instagram.svg" width="16" height="16"> </a> | <a href="https://nerdocs.atlassian.net/wiki/spaces/NER/overview?homepageId=65735"> <img src="https://cdn.worldvectorlogo.com/logos/confluence-1.svg" width="16" height="16"> </a>
+</p>
+
+We are a student-run organization at Northeastern University building all-electric formula-style race cars from scratch to compete in Formula Hybrid + Electric and Formula SAE.
+
+Our software team is a diverse group of software, computer, and electrical engineers committed to developing quality products and applications. The scope of the software team ranges from writing the code for real-time firmware control of critical automotive systems to developing a fully featured project management dashboard to maintain the timeline and organization required for building a racecar annually.
+
+## Our Active Projects
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
+| [Launchpad](https://github.com/Northeastern-Electric-Racing-Launchpad) (external org) | A git based educational system for understanding and honing basic skills in the world of software. | Git, Typescript, React, Express
+| **Application Software** | | |
+| [Argos](https://github.com/Northeastern-Electric-Racing/Argos) | A full stack data visualization and storage tool enabling real-time processing of car diagnostics for debugging and racetime information, as well as long term cloud management and querying of car statitistics. | Rust, MQTT, Prisma, Timescale/Postgres, Angular, Protobuf
+| [Nero](https://github.com/Northeastern-Electric-Racing/Nero-2.0) | An extensive and intuitive car dashboard. | QT, MQTT
+| **Firmware** | | | 
+| [Shepherd](https://github.com/Northeastern-Electric-Racing/ShepherdBMS-2)  | A from-scratch battery management system.  | C, FreeRTOS |
```

Comment:
shepherd is not FreeRTOS

Reply:
fixed. hopefully will be soon tho tons of rewriting has to get started lol.